### PR TITLE
fix: add krishnaUIDev

### PR DESCRIPTION
**Pull request by krishnaUIDev**
resolves #issueNumber

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/krishnaUIDev.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/24648588/208228038-8be44f01-b8c7-4418-8a82-901d9e9628b3.png",
+  "issueId": 1567,
+  "name": "KrishnaKanth Kondoju",
+  "username": "krishnaUIDev"
+}
```
Comment (Panquesito7) [inline profiles/krishnaUIDev.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/krishnaUIDev.json (hunk 1 above)
Comment (krishnaUIDev) [reply profiles/krishnaUIDev.json]: Updated

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**